pkgs/development/python-modules/monkeytype/default.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  wheel,
+  libcst,
+  mypy-extensions,
+}:
+
+buildPythonPackage rec {
+  pname = "monkeytype";
+  version = "23.3.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "Instagram";
+    repo = "MonkeyType";
+    rev = "v${version}";
+    hash = "sha256-DQ/3go53+0PQkhZcL2dX8MI/z4Iq7kTYd5EbacMNxT4=";
+  };
+
+  build-system = [
+    setuptools
+    wheel
+  ];
+
+  dependencies = [
+    libcst
+    mypy-extensions
+  ];
+
+  pythonImportsCheck = [
+    "monkeytype"
+  ];
+
+  meta = {
```

```suggestion
  ];

  nativeCheckInputs = [
    pytestCheckHook
  ];

  meta = {
```

Can you try enabling the tests please ?

-> Same for the other two packages.